src/browser/modules/D3Visualization/components/styled.tsx

```diff
@@ -371,3 +371,10 @@ export const PaneBodySectionHeaderWrapper = styled.div`
   display: flex;
   flex-direction: column;
 `
+
+export const StyledExpandValueButton = styled.button`
```

Comment:
Can't help but think we must have this exact component somewhere else also 🤔

Reply:
Found it! It's StyledShowMoreButton! Perhaps we could move that and reuse it? 🤔  Not sure it's the right move to de-duplicate when there's only two uses, let me know what you think

Reply:
yes I thought about it, but I think it is not necessary for now given the simplicity of the code. I think if we should do a general component it could be a ButtonStyledAsLink or something where also the color is sent it for a more useful reusability.